docs/multisite/languages/back_office_translations.md

```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which both install your JavaScript translations using the `BazingaJsTranslationBundle`, and clear the default SiteAccess cache.
+    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess if needed).
```

Comment:
```suggestion
    Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache (adapt SiteAccess if needed).
```
I thought we had a rule in vale for that 🤔

Reply:
Ok, I think it just generates a warning: https://github.com/ibexa/vale-styles/blob/main/styles/Ibexa/DontCapitalize.yml#L12